eks-cluster-setup/cluster.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: eksctl.io/v1alpha5
+availabilityZones:
+- eu-central-1a
+- eu-central-1b
+cloudWatch:
+  clusterLogging: {}
+iam:
+  vpcResourceControllerPolicy: true
+  withOIDC: false
+addons:
+- name: aws-ebs-csi-driver
+  serviceAccountRoleARN: "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
+  wellKnownPolicies:
+   ebsCSIController: true
+kind: ClusterConfig
+kubernetesNetworkConfig:
+  ipFamily: IPv4
+managedNodeGroups:
+- amiFamily: AmazonLinux2
+  iam:
+    withAddonPolicies:
+      ebs: true
+  instanceType: t2.2xlarge
+  labels:
+    alpha.eksctl.io/cluster-name: scrumptious-wardrobe-1684842095
+    alpha.eksctl.io/nodegroup-name: ng-d06bd84e
+  maxSize: 2
+  minSize: 2
+  name: ng-d06bd84e
+  releaseVersion: ""
+  ssh:
+    allow: true
+    publicKeyName: dektop-eu-central-1
```

Comment:
Is this something that will be generated or something people should have in their account?

Reply:
Its up on the user its not mandatory .... but yeah if you use it the key must exist .. I am mentioning it in the guide https://discourse.charmhub.io/t/deploying-charmed-mlflow-v2-to-eks/10913

Reply:
Added info to readme

Reply:
Okay, IMO you could leave the field with something like `{{ public_key_name }}` or `< public_key_name >` as people may forget about changing this before hand. I'll leave the decision up to you.

Reply:
Added :)